wgengine/magicsock/dontfrag_default.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build !linux && !darwin
```

Comment:
Given this platform constraint, are you going to introduce an exported function that other code will use to determine if PMTU can/can't be turned on for a particular platform?